httpserver/request_handler.cpp

```diff
@@ -76,9 +77,34 @@ Response RequestHandler::handle_wrong_swarm(const user_pubkey_t& pubKey) {
                     ContentType::json};
 }
 
+std::string computeMessageHash(const std::string& timestamp,
+                               const std::string& ttl,
+                               const std::string& recipient,
+                               const std::string& data) {
+
+    std::string payload;
+    payload.reserve(timestamp.size() + ttl.size() + recipient.size() +
+                    data.size());
+    payload += timestamp;
+    payload += ttl;
+    payload += recipient;
+    payload += data;
+
+    uint8_t hashResult[SHA512_DIGEST_LENGTH];

```

Comment:
```suggestion
    unsigned char hashResult[SHA512_DIGEST_LENGTH];
```
`SHA512` wants an `unsigned char*` argument, explicitly.

Reply:
Updated